components/notebook-controller/config/overlays/service-mesh/OWNERS

```diff
@@ -0,0 +1,10 @@
+# Each list is sorted alphabetically, additions should maintain that order
+approvers:
+- aslakknutsen
+- bartoszmajsak
+- cam-garrison
+
+reviewers:
+- aslakknutsen
+- bartoszmajsak
+- cam-garrison
```

Comment:
```suggestion
approvers: []
# - aslakknutsen
# - bartoszmajsak
# - cam-garrison

reviewers: []
# - aslakknutsen
# - bartoszmajsak
# - cam-garrison
```
Till we find when these contributor would be added to RHOAI, lets keep this empty.

Reply:
done